ui.apps/src/main/content/jcr_root/apps/etoolbox-backpack/assets/clientlibs/backpack/js/delete-action.js

```diff
@@ -12,36 +12,32 @@
  * limitations under the License.
  */
 
-(function(window) {
-    $(window).adaptTo("foundation-registry").register("foundation.collection.action.action", {
-        name: "backpack.delete",
+(function(Granite) {
+    $(window).adaptTo('foundation-registry').register('foundation.collection.action.action', {
+        name: 'backpack.delete',
         handler: function(name, el, config) {
+            let ui = $(window).adaptTo('foundation-ui');
+            let message = $('<div>');
 
-            let ui = $(window).adaptTo("foundation-ui");
-            let message = $("<div>");
+            const title = $('<p>').text(Granite.I18n.get('You are going to delete the following package:'));
+            const packageInfo = $('<p>').append($('<b>').text(config.data.packagePath));
+            message.append(title, packageInfo);
 
-            $("<p>").text(Granite.I18n.get("You are going to delete the following package:")).appendTo(message);
-            $("<p>").append($("<b>").text(config.data.packagePath)).appendTo(message);
-
-            ui.prompt(Granite.I18n.get("Delete"), message.html(), "notice", [{
-                text: Granite.I18n.get("Cancel")
-            }, {
-                text: Granite.I18n.get("Delete"),
+            const deleteBtn = {
+                text: Granite.I18n.get('Delete'),
                 warning: true,
-                handler: function () {
-                    deleteAction(config.data.packagePath);
-                }
-            }]);
+                handler: () => deleteAction(config.data.packagePath)
+            }
+
+            ui.prompt(Granite.I18n.get('Delete'), message.html(), 'notice', [{text: Granite.I18n.get('Cancel')}, deleteBtn]);
```

Comment:
(cosmetic) It does not make practical sense to wrap some of the labels into the `i18n` call. I know it was here before. Some people tend to wrap all the possible labels to kind of make sure that all the UI is translated. But as the project grows and more messages arrive, it goes out of hand. We are not managing i18n dictionaries in this project (and in the rest of EToolbox projects as well). So we might as well end up with a dialog prompt that has a text in English and the confirmation button in Arabic - what is it for?..

Reply:
Agree, what is your proposal? Can I retire i18n labels?